app/assets/javascripts/discourse/app/components/topic-dismiss-buttons.js

```diff
@@ -60,7 +61,10 @@ export default Component.extend({
 
   @discourseComputed("selectedTopics.length")
   dismissNewLabel(selectedTopicCount) {
-    if (selectedTopicCount === 0) {
+    const user = User.current();
```

Comment:
We should not be referring to the singleton here and can instead just refer to the current user service.